tests/local/conftest.py

```diff
@@ -0,0 +1,19 @@
+from browserstack.local import Local
+import pytest
+from selenium.webdriver.common.by import By
+import os
+BROWSERSTACK_ACCESS_KEY = os.environ['BROWSERSTACK_ACCESS_KEY']
+bs_local = Local()
+
+# You can also set an environment variable - "BROWSERSTACK_ACCESS_KEY".
+bs_local_args = { "key":  BROWSERSTACK_ACCESS_KEY if BROWSERSTACK_ACCESS_KEY else "BROWSERSTACK_ACCESS_KEY" }
```

Comment:
```suggestion
bs_local_args = { "key":  BROWSERSTACK_ACCESS_KEY or "YOUR_ACCESS_KEY" }
```

Reply:
Created one single conftest.py and made the change there. https://github.com/browserstack/pytest-browserstack/commit/29aa9a24727e13c6fc580c363fc6da5fb6db1249